kids/lib/screens/settings/applanguage.dart

```diff
@@ -16,24 +16,22 @@ class AppLanguageScreen extends HookConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    
     final design = DesignSystem.of(context);
     final bp = ResponsiveBreakpoints.of(context);
     final double basePadding = bp.smallerThan(TABLET) ? 24.0 : 48.0;
-    final selectedLanguageIndex = useState(0); 
+    final selectedLanguageIndex = useState(0);
 
-    List<ApplanguageListItem> buildItem() {
-      List<ApplanguageListItem> itemList = [];
-      for (int i = 0; i < appLanuageCodes.length; i++) {
-        itemList.add(ApplanguageListItem(
-          title: languages[appLanuageCodes[i]]!.nativeName,
+    List<AppLanguageListItem> buildItem() {
+      return appLanuageCodes.map((code) {
+        int index = appLanuageCodes.indexOf(code);
```

Comment:
Oh. you can simplify with `mapIndexed` if you need the index.
But this is also ok